### PR TITLE
[FLOC-3816] Release Flocker 1.9.0

### PR DESCRIPTION
**Pull request by zubron**
The build results for this branch: http://ci-live.clusterhq.com:8080/job/ClusterHQ-flocker/job/release-flocker-1.9.0/

One job is still failing:  http://ci-live.clusterhq.com:8080/job/ClusterHQ-flocker/job/release-flocker-1.9.0/job/run_lint/1/
This was decided to be an non-blocking issue for the release and was due to [FLOC-3869](https://clusterhq.atlassian.net/browse/FLOC-3869). This issue has now been fixed in master.

### CODE REVIEW

**Comment by zubron** [comment]
Forgot to add that the packages builds (which are created on Buildbot) are available here: http://build.clusterhq.com/boxes-flocker?branch=release/flocker-1.9.0

File: docs/releasenotes/index.rst [hunk 1]
```diff
@@ -9,18 +9,25 @@ You can learn more about where we might be going with future releases by:
...
 * The REST API now supports :ref:`conditional requests<conditional requests>` of the form "only create this dataset if the configuration hasn't changed since I last checked it", allowing for e.g. enforcement of metadata uniqueness.
-* The :ref:`Flocker Plugin for Docker<docker-plugin>` now solely relies on the metadata key ``"name"`` to find datasets.
-* Now supporting Ubuntu-15.10 instead of Ubuntu-15.04 for the flocker client.
+* Fixed a bug where :ref:`Flocker Plugin for Docker<docker-plugin>` could not create a dataset that had the same name as a previously deleted dataset.
+* Now supporting Ubuntu 15.10 instead of Ubuntu 15.04 for the Flocker client.
   See :ref:`installing-flocker-cli-ubuntu-15.10`.
-* Test being skipped (2 of them) in ``admin/test/test_release.py`` - once we have released the changes for ``15.10``, they don't need to be skipped anymore.
 * Added documentation for the :ref:`EMC VMAX <emc-dataset-backend>` driver.
+* Region and zone configuration for AWS is now validated before use.
+* Flocker now reports an error when busy EBS volumes can not be detached.
```
Comment (jml) [inline docs/releasenotes/index.rst]: Nit: s/can not/cannot/ ?

**Comment by jml** [comment]
LGTM. @davidoliverSP2 can you take a look before release?

**Comment by davidoliverSP2** [comment]
Looks good to me too - thanks @zubron 

I fixed a small issue which had misaligned the headings.

**Pull request merged**